package-drone/src/main/java/de/dentrassi/pm/jenkins/BuildData.java

```diff
@@ -21,7 +21,7 @@
 import hudson.model.ProminentProjectAction;
 
 @ExportedBean ( defaultVisibility = 999 )
-public class BuildData implements Action, Serializable, Cloneable, ProminentProjectAction
+public class BuildData implements Action, Serializable, ProminentProjectAction
```

Comment:
The correct fix is not remove interface but move clone method to public (as described in the clonable javadoc) and handle subclass of BuildData to invoke the mandatory `super.clone()`